<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MosaicDialog</class>
 <widget class="QDialog" name="MosaicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="20" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>fill colors</string>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QComboBox" name="fill_type_combo">
     <item>
      <property name="text">
       <string>none</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>keep borders</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>no borders</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QComboBox" name="combo_1">
     <item>
      <property name="text">
       <string>rotate around center</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rotate around vertex</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shift</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shift and flip</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>left</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>top</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>bottom</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QSpinBox" name="repeat_x">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>right</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>repeat horizontally</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QSpinBox" name="repeat_y">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="combo_2">
     <item>
      <property name="text">
       <string>rotate around center</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rotate around vertex</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shift</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shift and flip</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="21" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="17" column="3">
    <widget class="QLabel" name="validity">
     <property name="text">
      <string>validity</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>this specification is</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="QLineEdit" name="colors"/>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>fill type</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>repeat vertically</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="combo_3">
     <item>
      <property name="text">
       <string>rotate around center</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rotate around vertex</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shift</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shift and flip</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QComboBox" name="combo_4">
     <item>
      <property name="text">
       <string>rotate around center</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rotate around vertex</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shift</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shift and flip</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="MosaicSideEditFrame" name="point_edit_2">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>125</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>75</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="MosaicSideEditFrame" name="point_edit_3">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>125</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>75</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="MosaicSideEditFrame" name="point_edit_4">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>125</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>75</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="MosaicSideEditFrame" name="point_edit_1">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>125</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>75</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="clear_1">
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="clear_2">
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="clear_3">
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="clear_4">
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MosaicSideEditFrame</class>
   <extends>QFrame</extends>
   <header>mosaicsideeditframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MosaicDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MosaicDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
